sound/soc/sof/topology.c

```diff
@@ -407,6 +407,10 @@ static const struct sof_topology_token stream_tokens[] = {
 		offsetof(struct snd_sof_pcm, stream[0].d0i3_compatible)},
 	{SOF_TKN_STREAM_CAPTURE_COMPATIBLE_D0I3, SND_SOC_TPLG_TUPLE_TYPE_BOOL, get_token_u16,
 		offsetof(struct snd_sof_pcm, stream[1].d0i3_compatible)},
+	{SOF_TKN_STREAM_PLAYBACK_PAUSE_SUPPORTED, SND_SOC_TPLG_TUPLE_TYPE_BOOL, get_token_u16,
+		offsetof(struct snd_sof_pcm, stream[0].pause_supported)},
+	{SOF_TKN_STREAM_CAPTURE_PAUSE_SUPPORTED, SND_SOC_TPLG_TUPLE_TYPE_BOOL, get_token_u16,
+		offsetof(struct snd_sof_pcm, stream[1].pause_supported)},
```

Comment:
Do we need to have two tokens?

What if we just had one SOF_TKN_STREAM_PCM_PAUSE_SUPPORTED?

Presumably we already know the direction of the stream.

But maybe it's required because we don't have a means to apply the same value in the two directions?

Reply:
I have used the DoI3 support as template.
this is for the PCM device, it can be duplex or playback only or capture only.
It might be possible that only the playback is allowed to support pause while the capture is not in case of a duplex PCM device.

Reply:
ok